src/data/firmware/1/releases.json

```diff
@@ -1,4 +1,18 @@
 [
+	{
+		"required": false,
+		"version": [1, 10, 0],
+		"bootloader_version": [1, 10, 0],
+		"min_bridge_version": [2, 0, 25],
+		"min_firmware_version": [1, 6, 2],
+		"min_bootloader_version": [1, 5, 0],
+		"url": "data/firmware/1/trezor-1.10.0.bin",
+		"url_bitcoinonly": "data/firmware/1/trezor-1.10.0-bitcoinonly.bin",
+		"fingerprint": "595ba3e8e887cba185e03098f9538e18164f72f9fc82445e691abcd03e5cf0a4",
+		"fingerprint_bitcoinonly": "20a4068c34ff6dd7d8c510350409376cf7ea744ba668fdcf16da8f1d81fed289",
```

Comment:
I am not 100% I have done this correctly. I have used `trezorctl firmware-update -f` to find it out which is kind of lame. Please doube-check.